### ISSUE TITLE
Can't resolve 'material-ui-icons/BrokenImage' 

### ISSUE THREAD

**Issue by juliusspencer** [TeamWertarbyte/material-ui-image#17]
I'm on:
```
"material-ui-image": "^3.0.0-pre.6",
```

I can't compile and receive the following message:

```
./node_modules/material-ui-image/lib/components/Image/Image.js
Module not found: Can't resolve 'material-ui-icons/BrokenImage' in '/Users/julius/git/projects/freespace/web/freespace-web/node_modules/material-ui-image/lib/components/Image'
```

Might be something I'm doing wrong?
```
Uncaught Error: Cannot find module "material-ui-icons/BrokenImage"
    at webpackMissingModule (Image.js:56)
    at Object../node_modules/material-ui-image/lib/components/Image/Image.js (Image.js:56)
    at __webpack_require__ (bootstrap 20ff8bdf495a417520f5:669)
    at fn (bootstrap 20ff8bdf495a417520f5:87)
…
```

Nice component, thanks for any help!

**Comment by leMaik** [comment]
Whoops… I'll fix this. :scream:

**Comment by leMaik** [comment]
@juliusspencer The issue is fixed in v3.0.0-pre.7; we added `@material-ui/icons` as peer dependency. It already was required, but we forgot to add it. You need to install it in order to use this library, you'll get a warning from npm when it's missing.

**Comment by larinius** [comment]
Have exactly same error (after 5 years). And also other errors. This component creates more problems that solves.
As a result problem fixed by removing.
